src/heronarts/lx/parameter/BoundedParameter.java

```diff
@@ -23,14 +23,22 @@
  */
 public class BoundedParameter extends LXListenableNormalizedParameter {
 
-  public class Range {
+  static public class Range {
 
     public final double v0;
     public final double v1;
     public final double min;
     public final double max;
+    public final double vRange;
+    public final double range;
 
-    private Range(double v0, double v1) {
+    LXNormalizedParameter exponentObj;
```

Comment:
Feels slightly clunky to me, going to just make exponent a method parameter

Reply:
Felt weird to me too, I wasn't sure how to handle that one.  Thanks for cleaning it up.